tests/integration/cluster.go

```diff
@@ -1297,7 +1297,7 @@ func NewClusterV3(t testutil.TB, cfg *ClusterConfig) *ClusterV3 {
 	}
 	if !strings.HasPrefix(wd, os.TempDir()) {
```

Comment:
this check was new in 3.5.x and caused issues for me... t.TempDir() doesn't pass this prefix check on some operating systems

Reply:
fixed